src/Features/Core/Portable/ImplementInterface/AbstractImplementInterfaceService.CodeAction.cs

```diff
@@ -438,80 +428,6 @@ private static bool IsUnexpressibleTypeParameter(ITypeParameterSymbol typeParame
                 return generateInvisibly ? accessor : null;
             }
 
-            private SyntaxNode CreateThroughExpression(SyntaxGenerator generator)
```

Comment:
moved this helper to common location so it can be used by implement-abstract-class as well.